routes/flock.json

```diff
@@ -119,43 +119,282 @@
+        ],
+        "parallax": "/images/flock/goose4/lim.png"
+      },
+      {
+        "title": "Low Speed Drive",
+        "content": [
+          "Similar to the landing gear of an airplane, the low speed drive system serves as a fail-safe propulsion mechanism and a way to taxi the pod. Designed to operate at a speed of 3 m/s, the system consists of an electric hub motor that can be raised to reduce frictional force at higher speeds and lowered to safely drive the pod to the end of the track."
+        ],
+        "parallax": "/images/flock/goose4/low-speed-drive.png"
+      },
+      {
+        "title": "Braking",
+        "content": [
+          "Consisting of a 4-bar parallelogram linkage, these push the braking pads against the vertical section of the I-beam rail through a pneumatic actuator (air compression). This results in an effective and scalable braking system that can easily bring the pod to a safe stop from a high speed."
+        ],
+        "parallax": "/images/flock/goose4/braking.png"
+      },
+      {
+        "title": "Pneumatic Systems",
+        "content": [
+          "Used by the Low Speed Drive and Braking systems, the pneumatic system applies the brakes for deceleration and also controls the Low Speed Drive for taxiing and as a fail-safe, with both systems being independent of each other."
+        ],
+        "parallax": "/images/flock/goose4/pneumatic-system.png"
+      },
+      {
+        "title": "Frame",
+        "content": [
+          "As the main structural component of the GOOSE IV pod, the frame supports all other subsytems in a fixed position and transfers the overall propulsion force to the entire pod. Weighing only 9 kg, the frame is designed to withstand all acceleration, braking, vibration and structural loads during the run."
+        ],
+        "parallax": "/images/flock/goose4/frame.png"
+      },
+      {
+        "title": "Shell",
+        "content": [
+          "Designed with the focus on aesthetics, mass and aerodynamics, and inspired by the F-117 bomber, our smoked polycarbonate shell covers the entire pod while still allowing for its individual components to be shown and easily tweaked. Our GOOSE IV shell was also designed to stand out, with ample space to display our logo as well as promotional material."
+        ],
+        "parallax": "/images/flock/goose4/g4-shell.png"
+      },
+      {
+        "title": "Guidance",
+        "content": [
+          "Guiding the pod along the track, the guidance system provides lateral stability (so the pod doesn't tilt to one side) and controls vertical suspension. This system is split into the front and back of the pod, with the front holding the pod up and absorbing shocks from the track, and the back preventing the pod from derailing."
```

Comment:
Get rid of the "Guiding the pod along the track"

keep the sentence as "The guidance system ..."